### ISSUE TITLE
[BUG] When using Shizuku Installer, Droid-ify only installs to the profile Shizuku is running on.

### ISSUE THREAD

**Issue by Wave6677** [Droid-ify/client#718]
**Describe the bug**
When using Shizuku Installer, Droid-ify only installs to the profile Shizuku is running on.
For example, when Shizuku is installed in the main profile, and Droid-ify is installed on the "Island" profile of an Android device running Insular/Island/Calyx Work Profile etc, Droid-ify installs apps to the Main profile through Shizuku instead of the profile Droid-ify is running in
**To Reproduce**
1. Enable Shizuku Installer on Droid-ify with Shizuku in the opposite profile
2. Install any app
3. The app is now installed on the wrong profile

**Expected behavior**
Regardless of which profile Shizuku runs on, Droid-ify will install to the profile Droid-ify is running in

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy A54
 - OS: Android 14 and equivalent OneUI version

**Comment by Iamlooker** [comment]
Yes, and I don't know a proper fix for this. Use #995 to achieve the requirements maybe
